### ISSUE TITLE
[Screen Reader - Bot Framework Web Chat - Card Breakfast]: Screen reader is announcing incorrect role and state information for the 'Dismiss' button.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#5343]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
Screen reader users will be impacted if the incorrect role and state is announced for the controls. Screen reader users cannot know the current role, state of the control, and get confused and hence cannot effectively access the control.

### Note: 
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Breakfast word in type your message text box.
4. Navigate to the results.
5. Verify whether the correct role and state is announced or not.

### What do you expect?

Screen reader should announce correct role and state information for the 'Dismiss' button.
**Ex:** Screen reader should announce as 'Dismiss button'

### What actually happened?

 Screen reader is announcing incorrect role and state information for the 'Dismiss' button.
**Observation:** Screen reader (Narrator, NVDA and JAWS) is announcing as 'Dismiss toggle button off'

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/eed3988c-05ee-41f6-9204-50e7d71814ae)
https://github.com/user-attachments/assets/109a8d6f-7614-493d-b2e0-209370936c4e

### MAS reference

MAS 4.1.2 – Name, Role, Value

### WCAG reference

4.1.2 Name, Role, Value

### WAI-ARIA reference

https://www.w3.org/WAI/ARIA/apg/patterns/button/

### Note:
Same issue repro for all the buttons throughout the application.

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev2;#HCL;#Accessibility;#AILimited;#Benchmark;#BotFrameworkWebChat-Oct24;#MAS4.1.2;#Win11-Edge(Chromium);

**Comment by Kirank915** [comment]
@OEvgeny Could you please confirm if we need to route this bug as external to adaptive cards or will your team only fix this issue?

**Comment by OEvgeny** [comment]
This seems to be also authoring issue. I'm able to add a button without aria-pressed like so:

```json
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "Input.Date",
            "id": "test",
            "label": "Test"
        },
        {
            "type": "ActionSet",
            "actions": [
                {
                    "type": "Action.ToggleVisibility",
                    "title": "Action.ToggleVisibility",
                    "style": "positive"
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.3"
}
```
Can verify by going to https://adaptivecards.io/designer/ and pasting the above to the card payload editor located at the bottom
